### PR TITLE
Stable SigmoidCrossEntropyLoss.

### PR DESCRIPTION
**Pull request by t-k-**
`np.log(1 + np.exp(logits))' may have overflow, we can derive and
replace with its stable version.

### CODE REVIEW

**Comment by t-k-** [comment]
Before merge this one, please merge #8 first, because it has fixed the running problem.
